tests/test_repository_pypi.py

```diff
@@ -1,7 +1,9 @@
 import mock
 import pytest
+from mock import MagicMock, patch
 
-from piptools._compat.pip_compat import Link, Session, path_to_url
+from piptools._compat import PackageFinder, install_req_from_line
```

Comment:
```suggestion
from piptools._compat import PackageFinder
```

Use `from_line` instead as in the rest of the tests.